umbraco/Umbraco-CMS
# Reintroduce a favicon

Reintroduce a favicon. I took the default from asp.net core web apps.

### Test
- Run the templates and install a starter kit before first boot. 
- Verify static access are available
